lms/static/js/instructor_dashboard/membership.js

```diff
@@ -38,10 +42,8 @@ such that the value can be defined later than this assignment (file load order).
                 add_btn_label: 'Add Member',
                 add_handler: function() {}
             });
-            templateHtml = $('#member-list-widget-template').html();
-            edx.HtmlUtils.setHtml(
-                this.$container, window.Mustache.render(templateHtml, edx.HtmlUtils.HTML(memberListParams))
-            );
+            templateHtml = window.Mustache.render($('#member-list-widget-template').html(), memberListParams);
+            edx.HtmlUtils.setHtml(this.$container, edx.HtmlUtils.HTML(templateHtml));
```

Comment:
I have tried w/o that.
<img width="658" alt="screen shot 2016-10-25 at 9 40 33 am" src="https://cloud.githubusercontent.com/assets/16127998/19688299/2edc4bda-9a97-11e6-815c-3b50f8ea3dd5.png">